docker-compose.yml

```diff
@@ -213,7 +217,7 @@ services:
       zookeeper:
         condition: service_started
     ports:
-      - "9092:9092"
+      - "9094:9094"
```

Comment:
Keep it as 9092 only, that is the default.

Reply:
we are using 9092 for internal and for outside it's 9094